app/assets/javascripts/code/services/google/_map.js

```diff
@@ -1,5 +1,7 @@
 import _ from 'underscore';
 
+var Popup;
```

Comment:
I think we can get rid of this, create the class in another module, import it and use it normally